docs/client-side-usecases configuration.md

```diff
@@ -0,0 +1,105 @@
+
+_MyAccounts_ can provides many _use-cases_, that can be configured, which mean a hoster can offer a different way to discover available konnectors (aka present different _use-cases_).
+
+
+How to define a use-case?
+-------------------------
+
+All _use-cases_ are declared in a manifest, built-in the client app, at `client/app/contexts/<context>/index.json`. Its architecture is defined ine the _Architecture of the manifest_ section below.
+
+The `context` value is the one defines in which context _MyAccounts_ app run. There must be a `cozy` context, which is the default one available. Later, all partners can defines their own _use-cases_ by adding a new context, with a dedicated manifest and locales (see _Localization_ section below about translations).
+
+Adding a _use-case_ simply mean adding a new object into the _use-cases_ array, that provides the following keys (see the _Architecture_ section below for a complete list and informations):
+
+- `slug`: `<String>` the slugname for the use-case
+- `konnectors`: `<Array[Obj]>` an array of objects defining the konnectors
+- `figure`: `<String>` the picture file that illustrate the use-case
+- `color`: `<Color>` a color object that defines the color of the use-case
+- `default`: `<Bool>` a `true|false` value that defines this use-case as the default one
+- `important`: `<Bool>` should this `use-case` a recommended one?
+
+For its first release, the manifest will be built-in the client app, to get it ready to work out-of-the-box. With the stack v2, we should serve the manifest behind a dedicated endpoint, so we won't need to build a specific version of _MyAccounts_ each time we need to provides different _use-cases_.
+
+
+Default values
+--------------
+
+There's two kind of default values for a _use-case_:
+
+### the `default` key
+
+it defines if this _use-case_ is the default one. When we access the _use-cases_ screen (`/discover` URI), it offers a list of _use-cases_ screen, each one redirecting to a _use-case_ view (or the konnector if the _use-case_ only contains one).
+
+Sometimes, we need to directly open a _use-case_ screen without passing by the `/discover` view (this is the case when user access to _MyAccounts_ from the onboarding). In this case, this is the default _use-case_ which is displayed.
+
+⚠️ if the _use-cases_ array contains more than one _use-case_ with a default key set to `true`, then only the first one find in the array is considered as the default one.
+
+### The `incentive`
+
+When displaying a _use-case_ screen, a konnector can be highlighted first to incitate the user to first configure this one. Into the `konnectors` array, the _Konnector Object_ can define a `default` key at `true` to declares it as the _incentive_ one.
```

Comment:
It it the reference to `default` a typo ? Aren't we talking about `incentive` property ?